lib/core/src/sdk.rs

```diff
@@ -1213,6 +1240,17 @@ impl LiquidSdk {
         let mrh_addr_hash = sha256::Hash::hash(mrh_addr_str.as_bytes());
         let mrh_addr_hash_sig = keypair.sign_schnorr(mrh_addr_hash.into());
 
+        let webhook = self.persister.get_webhook_url()?.map(|url| ReverseWebhook {
+            url,
+            hash_swap_id: Some(true),
+            status: Some(vec![
+                RevSwapStates::InvoiceExpired,
+                RevSwapStates::SwapExpired,
+                RevSwapStates::TransactionConfirmed,
+                RevSwapStates::TransactionFailed,
+                RevSwapStates::TransactionMempool,
```

Comment:
We get both mempool and confirmed notification which means we will get two updates for the same payment.
Perhaps registering for only one of them based on whether we use zero conf or not for this swap?

Reply:
Sounds good. I know the TransactionConfirmed is skipped if it's 0-conf, but with two notifications for non 0-conf it would be spammy.